### DISCUSSION TITLE
cbc paralellization on ubuntu

### DISCUSSION THREAD

**Discussion by GiudittaDa** [coin-or/Cbc#460] [Q&A]
hello, i just want to understand if cbc on ubuntu got the flag –enable-cbc-parallel by default. i always used it on windows and i don't know if on linux is the same thing.
plus, can you help me write a easy script to see how many cores the solver is using? 
thank you a lot!

**Comment by svigerske** [comment]
It should be enabled by default on master (development version), but disabled by default on 2.10 and before.

**Reply by tkralphs** [reply]
We do not maintain the Ubuntu package, so it would be better to check directly with whoever maintains it to see what flag s are used when building. But if nothing else, you can just try to use multiple threads and see if it works.
